fwx.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "FWX",
+  "description": "FWX is a Permissionless Leverage Trading Platform for New Gem Tokens",
+  "imgUrl": "https://pbs.twimg.com/profile_images/1772244053326995456/XpthaJcx_400x400.jpg",
+  "missions": [
+    {
+      "name": "FWX_",
+      "description": "Complete a Liquidity Provision and Leverage Trading on FWX",
+      "imgUrl": "https://img5.pic.in.th/file/secure-sv1/mission1.jpg",
+      "tags": ["DeFi"],
+      "quests": [
+        {
+          "name": "Liquidity Provision",
+          "description": "Provide Liquidity AVAX on FWX",
+          "category": "DeFi",
+          "contract_address": "0x7F91272ff1A0114743D2df95F5905F9613Fd92b3",
+          "rewards": [
+            {
+              "type": "points",
+              "value": 60
+            }
+          ],
+          "tasks": [
+            {
+              "name": "Lend AVAX on official market on FWX",
+              "type": "LEND",
+              "description": "Complete an AVAX lend on FWX",
+              "contract_address": "0x7F91272ff1A0114743D2df95F5905F9613Fd92b3",
+              "method_id": "e2bbb158"
```

Method id missing "0x" prefix

```suggestion
              "method_id": "0xe2bbb158"
```